### DISCUSSION TITLE
MZK protocol

### DISCUSSION THREAD

**Discussion by Pat687426** [pascallanger/DIY-Multiprotocol-TX-Module#948] [General]
Is it possible to add MZK protocol ?
Thanks
Patrick

**Comment by pascallanger** [comment]
Need SPI dumps or access to the TX+RX to do anything...

**Reply by Pat687426** [reply]
Bonjour,
Merci de me préciser ce que sont les SPI dumps.
Je dispose d'un module tx uni à raccorder sur une prise ppm.
Patrick

**Reply by pascallanger** [reply]
Souder des fils sur le bus SPI entre le processeur et le composant RF qui sont à l'intérieur du module. Puis lancer des captures...

**Reply by Pat687426** [reply]
Désolé, je ne sais pas faire... Il faut être un as de soudure avec du matériel adapté pour ce faire.

J'ai posé la question pour MZK, car je dispose de ce protocole que j'avais installé sur une radio MPX MC4000, mais que je n'utilise plus.
Du coup, les modèles équipés de ce protocole ne volent plus !
C'est dommage, car ce système combine le FHSS et le DSSS, à priori assez sécurisant. Pour ma part je n'ai jamais eu de pb avec ce matériel.
